tests/unit_tests/test_game.py

```diff
@@ -630,6 +630,16 @@ async def test_to_dict(game, player_factory):
         "rating_min": game.displayed_rating_range.lo,
         "rating_max": game.displayed_rating_range.hi,
         "enforce_rating_range": game.enforce_rating_range,
+        "teams_ids": [
```

Comment:
Agh, I hate that this test still doesn't use hardcoded values. It's basically a reimplementation of the to_dict function.

Reply:
does this only test one (game, player_factory) or multiple combinations of the two ? in the later case how woud you hardcode that ?